pkg/pkg.ml

```diff
@@ -0,0 +1,8 @@
+#use "topfind"
+#require "topkg-jbuilder"
```

Comment:
cant do that, since `mirage-conduit` reverses the order, and I want to maintain backwards compat with the old packages (for now)